packages/sdk/src/swap/v2/types.ts

```diff
@@ -49,12 +49,12 @@ interface DepositFields {
 
 type ChunkInfo = {
   inputAmount: string;
-  intermediateAmount?: string;
-  outputAmount?: string;
+  intermediateAmount: string | undefined;
```

Comment:
consistency with other properties in this file - no strong opinion which syntax we should prefer